frontend/src/Components/ProfileForm.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react';
+import '../CSS/profile.css';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import { FaPencilAlt } from "react-icons/fa";
+
+const ProfileForm = ({user}) => {
+  const defaultImage = "https://uxwing.com/wp-content/themes/uxwing/download/peoples-avatars/person-profile-image-icon.png";
+
+  const [isDisabled, setEditable] = useState(true);
+
+  const handleEditable = () => setEditable(!isDisabled);
+
+  //Add function to save changes to profile.
+  const saveProfile = () => {};
+
+  return (
+    <div className='profile-form-container'>
+         
+        <form className='profile-form-form-container'>
+        <div class="profile-form-btn-container">
+            <a id="editBtn" className='profile-form-edit-button profile-form-save-text' onClick={handleEditable}>
+                {!isDisabled ? <span onClick={saveProfile}>Save</span> : <span id="editBtnText"><FaPencilAlt/></span>}
+            </a>
+        </div>
+        <div>
+          {/* Integrate PictureUpload component */}
+          { user.image === "" ? <img className="profile-form-image" src={defaultImage} alt="Profile"/> : <img className="profile-form-image" src={user.image} alt="Profile"/> }
+        </div>
+
+            <div className='profile-form-input-container'>
+              <input type='text' className="form-control profile-form-input-field" name="firstname" disabled={isDisabled} value={user.firstname} />
+              <input type='text' className="form-control profile-form-input-field"name="lastname" disabled={isDisabled} value={user.lastname} />
```

Comment:
If you really want to be consistent add a space between -field"name="lastname", but its not really necessary 😄